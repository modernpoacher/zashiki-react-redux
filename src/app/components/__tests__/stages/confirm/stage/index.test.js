import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'

import Stage from '@modernpoacher/zashiki-react-redux/app/components/stages/confirm/stage'

Enzyme.configure({ adapter: new Adapter() })

const MOCK_DEFINITION = {
  meta: {},
  elements: {}
}

describe('@modernpoacher/zashiki-react-redux/app/components/stages/confirm/stage', () => {
  it('renders', () => {
    const component = (
      <Stage
        definition={MOCK_DEFINITION}
        response={{}}
        onChange={jest.fn()}
        onSubmit={jest.fn()}
      />
    )

    expect(toJson(shallow(component)))
      .toMatchSnapshot()
  })
})
