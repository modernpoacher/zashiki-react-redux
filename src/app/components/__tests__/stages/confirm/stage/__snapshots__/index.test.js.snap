// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/stages/confirm/stage renders 1`] = `
<div
  className="confirm"
>
  <h2>
    Confirmation
  </h2>
</div>
`;
